packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -112,48 +112,27 @@
 					<AccordionTab header="Diagram">
 						<tera-model-diagram v-if="model" :model="model" :is-editable="false" />
 					</AccordionTab>
-					<template v-if="isPetriNet || isStockFlow">
```

Comment:
Im removing stockflow and regnet for now as we are designing for petrinet at the moment and this also wasnt a great way to do it.